.github/workflows/nightly.yml

```diff
@@ -18,14 +18,17 @@ jobs:
     steps:
     - name: Checkout Source
       uses: actions/checkout@v2
+      if: ${{ github.repository_owner == 'puppetlabs' }}
 
     - name: Activate Ruby 2.7
       uses: actions/setup-ruby@v1
```

Comment:
In https://github.com/voxpupuli/modulesync_config/pull/683 I figured out that there's [ruby/setup-ruby](https://github.com/ruby/setup-ruby) which can cache already using the proper methods. I'd recommend using it over `actions/setup-ruby`.